src/partials/welcome.ts

```diff
@@ -583,8 +583,7 @@ class WelcomeController {
             (error) => {
                 this.$log.error(this.logTag, 'Error state:', error);
                 // TODO: should probably show an error message instead
-                this.timeoutService.register(
-                    () => this.$state.reload(),
```

Comment:
I'd try to avoid unrelated style changes (as long as tslint doesn't complain) to retain `git blame` functionality.